pom.xml

```diff
@@ -355,11 +355,17 @@
       <artifactId>bcpkix-jdk15on</artifactId>
       <version>1.59</version>
     </dependency>
-      <dependency>
-          <groupId>com.amazonaws</groupId>
-          <artifactId>aws-java-sdk-sns</artifactId>
-          <scope>test</scope>
-      </dependency>
+    <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-sns</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.activation</groupId>
+      <artifactId>javax.activation</artifactId>
+      <version>1.2.0</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
`javax.activation` package was deprecated in JDK 1.9, so adding here to make the build success.